### DISCUSSION TITLE
cors issue

### DISCUSSION THREAD

**Discussion by codeMonkey-shin** [coder/code-server#7031] [Q&A]
I want to use the login API of the code server in my web app to obtain cookies and skip the login process to use the code server.

My web app's domain is domain.site, and the code server is on *.workspace.domain.site.

However, when I make an API request to obtain the cookies, a CORS error occurs. Is there a way to add the origin on the code server?

**Comment by code-asher** [comment]
Hmm...we have `--trusted-origins` but unfortunately I think this only affects web socket requests, so it would need to be changed to work with regular requests as well.

**Reply by codeMonkey-shin** [reply]
thanks :)
